module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -807,9 +807,12 @@ protected function getSimulatedStatus($id, ?array $patron = null)
     protected function setStatus(string $id, $holding = [], $append = true, $patron = null)
     {
         $session = $this->getSession($patron['id'] ?? null);
-        $i = isset($session->statuses[$id])
-            ? count($session->statuses[$id]) + 1 : 1;
-        $holding = array_merge($this->getRandomHolding($id, $i, $patron), $holding);
+
+        if (!$this->config['Holdings']['NoRandomHoldings'] ?? true) {
```

Comment:
The documentation says this defaults to false, but it defaults to true here. We should change it:


```suggestion
        if (!$this->config['Holdings']['NoRandomHoldings'] ?? false) {
```

(Though of course if you take my upcoming suggestion about renaming the setting, other adjustments will be needed here instead).